migrations/20190824182911-create-meal.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Meals', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
```

Comment:
Let's add the `allowNull` here as well just to make sure everything works if the other file is missing? @tnodland

Reply:
I'll add both to the next PR!

Reply:
By both I mean meals and foods

Reply:
👍